user_modules/shared_testcases/commonFunctions.lua

```diff
@@ -720,4 +722,55 @@ function commonFunctions:SDLForceStop(self)
 	os.execute("sleep 1")
 end
 
+---------------------------------------------------------------------------------------------
+--14. Function gets parameter from ini file
+---------------------------------------------------------------------------------------------
+function commonFunctions:read_parameter_from_ini(param_name)
```

Comment:
@dtrunov, I suggest get ini file name from parameter. So, this function will be more common.